### DISCUSSION TITLE
Phone into vr player

### DISCUSSION THREAD

**Discussion by Shadlt** [community/community#36859] [Other Feature Feedback, Questions, & Ideas] [answered]
How to make phone into vr player

**Comment by ossanna16** [comment, marked as answer]
Hey @Shadlt and thank you for your question! Can you please provide some more details so we can help you?

**Reply by Shadlt** [reply]
I  am trying to get my phone to work on vr applications with no device.

**Reply by ossanna16** [reply]
If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on VR. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by ariaesm** [comment]
you should learn and use AI
